README.md

```diff
@@ -68,7 +68,7 @@ Ladies Code Collective is hosted by Alex Morton - Episodes cover the coding jour
 
 Brought by Kody with Klossy, Sam and Jenny uncover stories of techsetters; hearing guests' career defining moments and unconventional paths in the tech world.
 
-- Topics: Tech, Changing Careers
+- Topics: Changing Careers, Tech
```

Comment:
Remove tech as the entire resources is about tech related items. It's sort of repetitive.